README.md

```diff
@@ -123,6 +123,8 @@ To build the tests, you will require the following:
 
 * [CMake](http://cmake.org), version 3.1.3 (3.2.3 for AppleClang) or later to be installed and in your PATH.
 
+NOTE: To run tests you need at least 3.8
```

Comment:
Here we can just replace this whole "To build the tests, you will require... NOTE: To run the tests" section with a simple
`To build the tests, you will require CMake version 3.8 or higher.`